client/src/LayoutComponents/Cohorts/Cohorts.js

```diff
@@ -1,37 +1,48 @@
-import React, { Component } from "react";
-import "./Cohorts.css";
-import CohortCard from "../../SharedComponents/CohortCard";
-import Slideshow from "../../SharedComponents/Slideshow";
-import axios from "axios";
+import React, { Component } from 'react';
+import './Cohorts.css';
+import CohortCard from '../../SharedComponents/CohortCard';
+import Slideshow from '../../SharedComponents/Slideshow';
+import axios from 'axios';
 
 class Cohorts extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cohortInfo: [],
+    };
+  }
   componentDidMount() {
     const cityName = this.props.match.params.city;
     axios
-      .get(`/cohorts/${cityName}`)
-      .then((response) => {
-        console.log(response.data);
+      .get(`/api/cohorts/${cityName}`)
+      .then((res) => {
+        const resData = res.data;
+        this.setState({ cohortInfo: resData });
+        console.log(this.state);
       })
       .catch((error) => {
         console.log(error);
       });
   }
   render() {
     const cityName = this.props.match.params.city;
-    // console.log(
-    //   this.props.history.location.state.data,
-    //   "this is cohorts page test "
-    // );
-
+    const { cohortInfo } = this.state;
+        
     return (
+      
       <div>
-        <h1 className="title">{cityName} Cohorts</h1>
-        <div className="cohort">
-          <div className="coh_card">
-            <CohortCard />
-          </div>
-          <Slideshow />
+      {!cohortInfo.length? <h1>loading</h1>: 
+      <div>
+        <h1 className='title'>{cityName} Cohorts</h1>
+          {cohortInfo.map((e) =>(
```

Comment:
@bayan-404 It might be a good idea to have a more descriptive variable name than `e` here.